build-tools/packages/build-cli/test/library/branches.test.ts

```diff
@@ -129,6 +129,12 @@ describe("generateReleaseBranchName", () => {
 		assert.equal(actual, expected);
 	});
 
+	it("Fluid RC version scheme", () => {
+		const actual = generateReleaseBranchName(MonoRepoKind.Client, "2.0.0-rc.1.0.0");
```

Comment:
Also test that --branchCheck accepts RC branches?

Reply:
That's not a test that we can easily write. Doing so would require us to mock the whole repo, becuuse the command queries the repo itself to check the current branch.

However, unit testing these functions is 99% of what's needed. Most of the command logic is just glue between individually testable functions.